opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicMetricExporter.java

```diff
@@ -209,19 +207,25 @@ public NewRelicMetricExporter build() {
         return new NewRelicMetricExporter(
             telemetryClient, commonAttributes, timeTracker, new MetricPointAdapter(timeTracker));
       }
-      MetricBatchSenderBuilder builder = SimpleMetricBatchSender.builder(apiKey);
-      builder.secondaryUserAgent("NewRelic-OpenTelemetry-Exporter", null);
-      if (enableAuditLogging) {
-        builder.enableAuditLogging();
-      }
+      SenderConfiguration.SenderConfigurationBuilder builder =
+          MetricBatchSenderFactory.fromHttpImplementation(OkHttpPoster::new)
+              .configureWith(apiKey)
+              .secondaryUserAgent("NewRelic-OpenTelemetry-Exporter")
+              .auditLoggingEnabled(enableAuditLogging);
+
       if (uriOverride != null) {
         try {
-          builder.uriOverride(uriOverride);
+          if (uriOverride.getPath() != null && !uriOverride.getPath().isEmpty()) {
```

Comment:
Could use that new helper method for this familiar bit.  😺

Reply:
yeah, wrong module, though.

Reply:
bummer.  well, one day we'll fix it up in the telemetry sdk and this will be irrelevant.  So yeah.